tests/cases/compiler/importShouldNotBeElidedInDeclarationEmit.ts

```diff
@@ -0,0 +1,63 @@
+// @declaration: true

```

Comment:
Use a minimal test that reproduces the error -- looking at this test I would have no idea what it's trying to test..

Reply:
We should always strive for the smallest reproducible steps for a failure.

Reply:
It is now minimal